apps/app/src/app/core/services/settings.service.ts

```diff
@@ -1,16 +1,70 @@
+import { HttpParams } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 import { Application } from '@armonik.admin.gui/armonik-typing';
 
 @Injectable({
   providedIn: 'root',
 })
 export class SettingsService {
+  seqEndpoint: string | null = null;
   currentApplications: Set<Application['_id']>;
 
   constructor() {
     this.currentApplications = new Set(this.getCurrentApplicationsFromStore());
   }
 
+  /**
+   * Verify if Seq is up and running
+   *
+   * @returns True if Seq is up and running, false otherwise
+   */
+  isSeqUp(): boolean {
+    return this.seqEndpoint !== null;
+  }
+
+  /**
+   * Generate Seq url
+   *
+   * @param query Query to add to the url
+   *
+   * @returns Seq url
+   */
+  generateSeqUrl(query: { [key: string]: string }): string {
+    if (!this.seqEndpoint) return '';
+
+    // If seqEndpoint doesn't contains a slash at the end, add it
+    if (this.seqEndpoint.slice(-1) !== '/') {
+      this.seqEndpoint += '/';
+    }
+
+    // If seqEndpoint doesn't contains http:// or https://, add it
+    if (
```

Comment:
Ici tu pourrais plutôt faire : 
 `   try {
      const url = new URL(this.seqEndpoint);
       if(url.protocol !== "http:" && url.protocol !== "https:") {
            this.seqEndpoint = 'http://' + this.seqEndpoint;
       }
    } catch (e) {
      this.seqEndpoint = `\``http://${this.seqEndpoint}`\``;
    }
`

Reply:
done